src/js/components/Select/SelectContainer.js

```diff
@@ -359,6 +359,10 @@ class SelectContainer extends Component {
 
     const customSearchInput = theme.select.searchInput;
     const SelectTextInput = customSearchInput || TextInput;
+    const selectOptionsStyle = {
+      ...theme.select.options.box,
+      ...theme.select.options.container,
```

Comment:
what will happen if we'll have same props defined on both `theme.select.options.box` & `theme.select.options.container`?

Reply:
`select.options.container` will "win". I have included a test for this scenario.